src/test/integration/storage/test_db_interface_compare.py

```diff
@@ -1,28 +1,26 @@
+# pylint: disable=attribute-defined-outside-init,protected-access
 import gc
-import unittest
 from time import time
 
 import pytest
 
 from helperFunctions.config import get_config_for_testing
-from storage.MongoMgr import MongoMgr
 from storage.db_interface_admin import AdminDbInterface
 from storage.db_interface_backend import BackEndDbInterface
 from storage.db_interface_common import MongoInterfaceCommon
-from storage.db_interface_compare import (
-    CompareDbInterface, FactCompareException
-)
+from storage.db_interface_compare import CompareDbInterface, FactCompareException
+from storage.MongoMgr import MongoMgr
 from test.common_helper import create_test_firmware
 
 
-class TestCompare(unittest.TestCase):
+class TestCompare:
 
     @classmethod
-    def setUpClass(cls):
+    def setup_class(cls):
         cls._config = get_config_for_testing()
         cls.mongo_server = MongoMgr(config=cls._config)
 
-    def setUp(self):
+    def setup(self):
```

Comment:
Is this still functional? `setUp` is a special function name defined in unittest
https://docs.python.org/3/library/unittest.html

Reply:
`setup`, `teardown`, `setup_class` and `teardown_class` are the equivalent functions in `pytest`